src/app/sitemap.js

```diff
@@ -0,0 +1,73 @@
+import { getBlogByAllFilter } from "@/services/api/blogAPI";
+import { getConferenceByAllFilters } from "@/services/api/conferenceAPI";
+import { getHackathonByAllFilters } from "@/services/api/hackathonAPI";
+import { getNewsletterByAllFilter } from "@/services/api/newsletterAPI";
+import { getPodcastByAllFilter } from "@/services/api/podcastAPI";
+import { getYoutubeByAllFilter } from "@/services/api/youtubeAPI";
+
+const sitemap = async () => {
+  const BASE_URL = "https://devresources.info";
+
+  const conferences = await getConferenceByAllFilters(
+    undefined,
```

Comment:
Why do we need to pass these many undefined.. if you do not pass anything.. every argument is still undefined. @prachi-sahu-codes